Providers/nxOMSAutomationWorker/AzureAutomation-Worker-for-Linux/scripts/certgenerator.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python
+
+"""Script to generate certificate and key file."""
+
+from OpenSSL import crypto
+from socket import gethostname
+import argparse
+
+
+def generate_cert_and_key(cert_name, key_name):
```

Comment:
Why do we need this. We will be using the oms agent cert for the initial registration and auth against JRDS.